README.md

```diff
@@ -273,6 +290,32 @@ associative array in the first argument:
 </div>
 ```
 
+You can also pass parameters to actions:
+
+```twig
+<div {{ stimulus_action('controller', 'method', 'click', { 'count': 3 }) }}>Hello</div>
+
+<!-- would render -->
+<div data-action="click->controller#method" data-controller-count-param="3">Hello</div>
+```
+
+You can chain calls if you prefer as there's also a `stimulus_action` filter:
+
+```twig
+<div {{ stimulus_action('controller', 'method', 'click', { 'count': 3 })|stimulus_action('other-controller', 'test') }}>
+    Hello
+</div>
+```
+
+You can also retrieve the generated attributes as an array, which can be helpful e.g. for forms:
+
+```twig
+{{ form_start(form, { 'attr': stimulus_action({
+    'controller': 'method',
+    'other-controller': ['method', {'resize@window': 'onWindowResize'}]
+).toArray() }) }}
```

Comment:
Use a single-action example... and let's use a field as an example (e.g. `form_row(form.password)`) as an example, perhaps calling some `checkPasswordStrength` method).